Settings on query string

lolivei opened the pull request:
Hi, Jaden! How are you doing?

Really nice tool!
If you don't mind, I created this PR that adds support for providing the parameters on the URL query string.
It keeps supporting the key provided on the hash part of the URL.

Thanks!

jaden reviewed (changes requested):
Cool, thanks for the additions.

I have a few minor requests.

1) Could you add back the hash in the URL example in the README so it's documented?
2) Could you rename getSettings to getQueryParameters so it's easier to tell what the method does?
3) I'd like to maintain support for older browsers if possible and it looks like [URLSearchParams](https://caniuse.com/urlsearchparams) doesn't have great support in earlier browsers. Do you happen to know of a polyfill or an alternative? If not, it's not a big deal.

Thanks again,

Dan

lolivei commented:
Hi Dan,

Changes made as requested.
Added a helper function in order to parse the query string that doesn't rely on URLSearchParams.
I also changed the README to better describe both ways of providing the private key.

Thanks for considering the additions.

Lucas

jaden reviewed (approved):
The changes look great, thanks again for your contribution 🥇
